modin/experimental/torch/tests/test_dataloader.py

```diff
@@ -41,34 +55,24 @@ def _test_torch_dataloader(pandas: ModuleType):
             "POPULATION",
             "PRICE",
         ],
+        sampler=sampler_cls,
     )
+
+    outputs = []
     for batch in loader:
-        assert batch.shape[0] <= 16, batch.shape
+        assert batch.shape[0] <= batch_size, batch.shape
         assert batch.shape[1] == 6, batch.shape
 
+        outputs.append(batch)
 
-def test_torch_dataloader():
-    _test_torch_dataloader(pd)
-    _test_torch_dataloader(mpd)
+    return outputs
 
 
-def test_random():
-    df = pd.read_csv(
-        "https://raw.githubusercontent.com/ponder-org/ponder-datasets/main/USA_Housing.csv"
-    )
-    loader = ModinDataLoader(
-        df,
-        batch_size=16,
-        features=[
-            "AVG_AREA_INCOME",
-            "AVG_AREA_HOUSE_AGE",
-            "AVG_AREA_NUM_ROOMS",
-            "AVG_AREA_NUM_BEDROOMS",
-            "POPULATION",
-            "PRICE",
-        ],
-        sampler=RandomSampler,
-    )
-    for batch in loader:
-        assert batch.shape[0] <= 16, batch.shape
-        assert batch.shape[1] == 6, batch.shape
+@pytest.mark.parametrize("sampler_cls", [RandomSampler, SequentialSampler])
+def compare_dataloaders(sampler_cls: Type[Sampler]):
```

Comment:
Is this even being called? I think pytest won't consider this as a test because of its name not starting with `test_`...

Reply:
Oops, my bad.